ROMFS/px4fmu_common/init.d/4030_3dr_solo

```diff
@@ -5,11 +5,7 @@
 # @type Quadrotor x
 # @class Copter
 #
-# @board px4fmu-v2 exclude
-# @board px4fmu-v4 exclude
-# @board px4fmu-v4pro exclude
-# @board px4fmu-v5 exclude
-# @board aerofc-v1 exclude
+# @board px4fmu-v3 include
```

Comment:
@dagar Flash is overflowing now with this change - which is a hint to me that it is not working as designed.